datamodel_code_generator/model/template/Union.jinja2

```diff
@@ -0,0 +1,10 @@
+{%- if description %}
+# {{ description }}
+{%- endif %}
+{%- if fields|length > 1 %}
+{{ class_name }}: TypeAlias = Union[
```

Comment:
`TypeAlias`  is good for defining types.
We would apply `TypeAlias` to other output models like TypedDict and dataclasses next time.